jekyll/_cci2/gh-bb-integration.md

```diff
@@ -296,20 +296,24 @@ by following this workaround:
 1. In the CircleCI application,
 go to your project's settings.
 
-2. Navigate to the **Checkout SSH Keys** page.
+2. Navigate to the **SSH Keys** page.
 
-3. Right-click the **Create `<username>` user key** button
-and select the **Inspect** option to open the browser inspector.![]( {{ site.baseurl }}/assets/img/docs/bb_user_key.png)
+3. - **For users of the new CircleCI UI:** Right-click the **Add User Key** button and select the **Inspect** option to open the browser inspector.![]( {{ site.baseurl }}/assets/img/docs/new_bb_user_key.png)
```

Comment:
Would you mind just removing references to using the new UI? Now that everyone is using it we can just go with the steps as they are - in this case just remove: "**For users of the new CircleCI UI:**"